tests/Elasticsearch/Tests/ConnectionPool/Selectors/RoundRobinSelectorTest.php

```diff
@@ -29,19 +30,32 @@ public function testTenConnections()
     {
         $roundRobin = new Elasticsearch\ConnectionPool\Selectors\RoundRobinSelector();
 
-        $mockConnections = array();
-        foreach (range(0, 10) as $index) {
-            $mockConnections[$index] = $this->getMockBuilder('\Elasticsearch\Connections\CurlMultiConnection')
+        $mockConnections = [];
+        foreach (range(0, 9) as $index) {
+            $mockConnections[$index] = $this->getMockBuilder(ConnectionInterface::class)
```

Comment:
I replaced the string classname (btw. non-existent since ES-PHP 2.0 ;-) ) with `::class` reference. It will fail as soon as you remove the class.